src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -24,10 +24,12 @@
 
     <!-- Tells CPS which target to run for the Package solution build type -->
     <PackageAction Condition="'$(PackageAction)' == ''">Pack</PackageAction>
+  </PropertyGroup>
 
+  <ItemGroup>
     <!-- Disable Build Acceleration for VSIX projects -->
-    <AccelerateBuildsInVisualStudio Condition="'$(IsVsixProject)' == 'true'">false</AccelerateBuildsInVisualStudio>
```

Comment:
This never really worked property. Turns out that `IsVsixProject` is our own property, not one defined in `Microsoft.VSSDK.BuildTools`'s `.props` or `.targets`.